### DISCUSSION TITLE
How to earn Galaxy Brain badge?

### DISCUSSION THREAD

**Discussion by Dnkfff** [community/community#106947] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/56574536/490f2bd9-f57c-4e57-a9f2-b565b48d9954)

I was trying almost everything, it is way tougher that I actually thought

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Dnkfff, 

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team

I also wanted to share: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers.
